src/python/WMComponent/JobUpdater/JobUpdaterPoller.py

```diff
@@ -93,11 +93,19 @@ def algorithm(self, parameters=None):
             msg += "transactions postponed until the next polling cycle\n"
             msg += str(ex)
             logging.exception(msg)
+        except CouchError as ex:
+            if ex.status is None and ex.reason is None:
```

Comment:
Looking at the code, I wonder if we can ever have ex.reason=None? It seems reason would always have a non None value (unless we explicitly construct it with None)

Reply:
Thanks for the review, Alan.
I am not sure but this particular error contains reason is None

```
2018-04-14 09:24:41,838:139750312437504:ERROR:JobUpdaterPoller:Caught unexpected exception in JobUpdater: CouchError - reason: None, data: {'stale': 'update_after', 'reduce': 'false'} result: None
Traceback (most recent call last):
```
I just don't want to be too general for catching errors at least for now.